src/lib/Twig/Components/SingleInputFieldTrait.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\DesignSystemTwig\Twig\Components;
+
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\PropertyAccess\Exception\InvalidTypeException;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+/**
+ * @phpstan-type AttributeMap array<string, scalar>
+ */
+trait SingleInputFieldTrait
```

Comment:
tbh, this seems like a little to bit and to specified for a trait, is there a way to make service from it and inject it where needed?